patch/u-boot/u-boot-rockchip64/board_mamopct4/u-boot-rk-rk3399-usb-start-nanopc-t4.patch

```diff
@@ -0,0 +1,28 @@
+--- a/configs/nanopc-t4-rk3399_defconfig
```

Comment:
Why adding this patch in a folder specific to `nanopct4`?
It is not erroneous but IIRC these board specific folders were useful when you had some conflicting changes between the boards.
This change does not seem conflicting with anything so it may as well live in a main `u-boot-rockchip64` folder.

The other thing is that this patch will most probably not apply at all as the folder name is wrong: `mamopct4` vs `nanopct4` ;-)